src/coreclr/src/inc/cor.h

```diff
@@ -1825,27 +1825,6 @@ DECLARE_INTERFACE_(IMetaDataInfo, IUnknown)
         DWORD *       pdwMappingType) PURE; // [out] Type of file mapping (code:CorFileMapping).
 };  // class IMetaDataInfo
 
-
-//-------------------------------------
-//--- IMetaDataWinMDImport
-//-------------------------------------
-// {969EA0C5-964E-411B-A807-B0F3C2DFCBD4}
-EXTERN_GUID(IID_IMetaDataWinMDImport, 0x969ea0c5, 0x964e, 0x411b, 0xa8, 0x7, 0xb0, 0xf3, 0xc2, 0xdf, 0xcb, 0xd4);
```

Comment:
Let's be careful of all these interface changes and updates. If all of them are internal then updates are fine, but we need to make sure that they can't be implemented by external people without a GUID change or else vtables will be in a bad state.